CodeEdit/Features/CommandsOverlay/ViewModels/CommandsOverlayViewModel.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  CommandsOverlayViewModel.swift
+//  CodeEdit
+//
+//  Created by Alex on 25.05.2022.
+//
+
+import Combine
+import SwiftUI
+
+/// Simple state class for commands overlay view. Contains currently selected command,
+/// query text and list of filtered commands
+final class CommandsOverlayViewModel: ObservableObject {
+
+    @Published
+    var commandQuery: String = ""
+
+    @Published
+    var filteredMenuCommands: [CodeEditCommand] = []
+
+    var menubarWatcher: AnyCancellable?
```

Comment:
Because this value is assigned in the constructor, it doesn't have to be nil right? Or does MergeMany return an optional?